tests/phpunit/tests/html-api/wpHtmlTagProcessor-bookmark.php

```diff
@@ -299,15 +299,23 @@ public function test_bookmarks_complex_use_case() {
 	public function test_updates_bookmark_for_additions_after_both_sides() {
```

Comment:
You might wanna add the `@ticket XXXXXX` reference for this change to this test's docblock as well 🙂

Reply:
Added. Is it fine to add the explanation after the ticket id or will that throw off some parser somewhere? I can remove the explanation if it's a problem.

Reply:
It looks as if no other test cases are using an explanation after the ticket id, so maybe better remove it for consistency's sake.